src/layouts/Workspace.jsx

```diff
@@ -20,6 +19,7 @@ import { prettifyPageFileName } from '../utils';
 
 // Import hooks
 import useSiteColorsHook from '../hooks/useSiteColorsHook';
+import useRedirectHook from '../hooks/useRedirectHook';
```

Comment:
unused import

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/359/commits/b910e4654fdd2a7fc17860197412921b364d5be3